libuavcan/include/libuavcan/util/math.hpp

```diff
@@ -6,13 +6,13 @@
  */
 /** @file */
 
-#ifndef UAVCAN_UTIL_MATH_HPP_INCLUDED
-#define UAVCAN_UTIL_MATH_HPP_INCLUDED
+#ifndef LIBUAVCAN_UTIL_MATH_HPP_INCLUDED
+#define LIBUAVCAN_UTIL_MATH_HPP_INCLUDED
 
-#include "uavcan/uavcan.hpp"
+#include "libuavcan/uavcan.hpp"
 #include <cstdlib>
 #include <cmath>
-#include "uavcan/introspection.hpp"
+#include "libuavcan/introspection.hpp"
```

Comment:
This note is not related to this PR but it seems important: you seem to be assuming here that the included path is located inside one of the include look-up directories since it's not reachable relatively. Quotes should be replaced with angle brackets (here and elsewhere).

Reply:
<> first checks system paths whereas "" first checks local paths then system. The latter seems more correct for a header-only library since we do not expect it to be available on a system.

Reply:
I see,